modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -89,6 +93,48 @@ finished.
 > the codemod, as its resulting formatting (spacing, quotes, etc.) may not match your project
 > conventions.
 
+## Component Promotions
+
+### Table (Preview)
+
+**PR:** [#2600](https://github.com/Workday/canvas-kit/pull/2600)
+
+We've promoted the `Table` component in preview to main.
+
+## Removals
+
+### Table (Main)
+
+**PR:** [#2600](https://github.com/Workday/canvas-kit/pull/2600)
+
+We've removed the `Table` component in main and replaced it with the `Table` component from preview.
+This will consist of a new API since it is now a compound component.
+
+> `rowState` prop is no longer a part of the `Table` component.
+
+See below for how these components will migrate:
+
+|      v10       |         v11        |
+| -------------- | ------------------ |
+| `<Table/>`     | `<Table/>`         |
+|      `n/a`     | `<Table.Caption/>` |
+|      `n/a`     | `<Table.Head/>`    |
+|      `n/a`     | `<Table.Body/>`    |
+| `<TableRow/>`  | `<Table.Row/>`     |
+|      `n/a`     | `<Table.Header/>`  |
+|      `n/a`     | `<Table.Cell/>`    |
+|      `n/a`     | `<Table.Footer/>`  |
+
+In v10, `Table` and `TableRow` were the pieces that would be used to build a Table.
+In v11, `Table` is now a compound component that utilizes all the pieces of an html
+[Table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table) element to build a Table.
+
+
+Please refer to the [Table examples](https://workday.github.io/canvas-kit/?path=/story/components-containers-table--basic) for how
+to implement new tables.
+
+> **NOTE: This will have to be updated manually as the API is too different for a codemod.**
+
```

Comment:
RC: Let's remove this line. 
```suggestion
```

Reply:
I think it's worth noting that if you're importing table from preview, make sure to change the import. There's not codemod for this since the change to compound component might be too complex, but worth notion about the import.